i18n.js

```diff
@@ -22,7 +22,10 @@ define('i18n', ['ajax'], function(ajax) {
 		if (!version || !message || JSON.parse(message).version != version) {
 			updateLocalI18n(key, this.domain);
 		}
-		return JSON.parse(localStorage.getItem(key)).message;
+		if(localStorage.getItem(key)) {
+			return JSON.parse(localStorage.getItem(key)).message;
```

Comment:
You should extract `localStorage.getItem(key)` to avoid multiple gets for the same value.

@rapahaeru @mottam